pkgs/by-name/ho/houdini/package.nix

```diff
@@ -76,7 +76,7 @@ buildFHSEnv rec {
       "houdini/sbin/sesinetd"
     ];
   in ''
-    WRAPPER=$out/bin/${name}
+    WRAPPER=$out/bin/${pname}
```

Comment:
This is [`pname` misuse](https://github.com/NixOS/nixpkgs/issues/277994). Might be worth fixing it in this PR since the line is changed anyway.